### PR TITLE
Remove deadcode and add OWNERs files

### PR DESCRIPTION
**Pull request by jlewi**
Remove dead code
* gatekeeper - This was for GCP basic auth which is no longer supported
* https-redirect - This was to redirect http to https and was part of GCP
  click to deploy which has been turned down.
* image-releaser - This is defunct code for CD of docker images. This
  has been replaced by Tekton scripts
* kflogin - This was the login page for basic auth

Add OWNERs files for applications related to GCP

* echo-server - A very simple web app used to debug IAP
* ingress-setup-image - Docker image that we use to run some of our
  IAP setup scripts in
* private-utils - Docker image for mirror'ing images for private GKE
* static-config-server - Not sure what this is used for.

### CODE REVIEW

**Comment by Bobgy** [comment]
Thanks @jlewi !
/lgtm
/approve

**Comment by swiftdiaries** [comment]
Deleting code is fun :) 
/lgtm

File: components/image-releaser/app.yaml [hunk 1]
```diff
@@ -1,24 +0,0 @@
-apiVersion: 0.1.0
```
Comment (Jeffwan) [inline components/image-releaser/app.yaml:1]: Image releaser has been migrated to tekton? Or kubeflow won't release images officially?

Comment (Jeffwan) [reply components/image-releaser/app.yaml:1]: Just notice this is mentioned in the description.

**Comment by Jeffwan** [comment]
looks good

**Pull request merged**